topic-operator/src/main/java/io/strimzi/operator/topic/TopicSerialization.java

```diff
@@ -206,6 +230,26 @@ public static Topic fromTopicMetadata(TopicMetadata meta) {
         return builder.build();
     }
 
+    /**
+     * Create a Topic to reflect the given TopicMetadata.
+     */
+    public static Topic fromTopicMetadata(TopicMetadata meta, ObjectMeta k8sMeta) {
```

Comment:
If you really need this overloaded method should should make the old one (above) just delegate to this new one passing a null `k8sMeta`. By copying and modifying it you've created a load of duplicated code which we would have to maintain.